### DISCUSSION TITLE
Improve the accuracy of text detection

### DISCUSSION THREAD

**Discussion by leviethung2103** [PaddlePaddle/PaddleOCR#10403] [Q&A]
Hello there,
I have two questions

1. How to improve the text detection for small images ?  What are the configuration parameters need to change during the training ? 

2. Currently, I used the mobilenetv3 large x0.5 to train the text detection model. However, the results are not good. 

![image](https://github.com/PaddlePaddle/PaddleOCR/assets/16741872/53a86568-f66e-4036-b3c2-6448dbd8962b)

Here is my config file.
```
Global:
  use_gpu: true
  epoch_num: 50
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ./output/db_mv3/
  save_epoch_step: 1200
  # evaluation is run every 2000 iterations
  eval_batch_step: [0, 1000]
  cal_metric_during_train: False
  pretrained_model:
  checkpoints:
  save_inference_dir:
  use_visualdl: False
  infer_img: doc/imgs_en/img_10.jpg
  save_res_path: ./output/det_db/predicts_db.txt

Architecture:
  model_type: det
  algorithm: DB
  Transform:
  Backbone:
    name: MobileNetV3
    scale: 0.5
    model_name: large
  Neck:
    name: DBFPN
    out_channels: 256
  Head:
    name: DBHead
    k: 50

Loss:
  name: DBLoss
  balance_loss: true
  main_loss_type: DiceLoss
  alpha: 5
  beta: 10
  ohem_ratio: 3

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.0001
    warmup_epoch : 2
  regularizer:
    name: 'L2'
    factor: 0.00001

PostProcess:
  name: DBPostProcess
  thresh: 0.3
  box_thresh: 0.6
  max_candidates: 1000
  unclip_ratio: 1.5

Metric:
  name: DetMetric
  main_indicator: hmean

Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./data 
    label_file_list:
      - ./data/train_lpd_label.txt
    ratio_list: [1.0]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - DetLabelEncode: # Class handling label
      - IaaAugment:
          augmenter_args:
            # - { 'type': Fliplr, 'args': { 'p': 0.5 } }
            - { 'type': PerspectiveTransform, 'args': { 'scale' : [0, 0.05] } } 
            - { 'type': Affine, 'args': { 'rotate': [-5, 5] } }
            - { 'type': Resize, 'args': { 'size': [0.9, 1] } }
            # - { 'type': Resize, 'args': { 'size': [0.5, 3] } }
            - { 'type': GaussianBlur, 'args': { 'sigma': [0.0, 1.0] } }
            - { 'type': GammaContrast, 'args': { 'gamma': [0.5, 1.0] } }
      - EastRandomCropData:
          size: [480, 480]
          max_tries: 50
          keep_ratio: true
      - MakeBorderMap:
          shrink_ratio: 0.4
          thresh_min: 0.3
          thresh_max: 0.7
      - MakeShrinkMap:
          shrink_ratio: 0.4
          min_text_size: 8 
      - NormalizeImage:
          scale: 1./255.
          mean: [0.485, 0.456, 0.406]
          std: [0.229, 0.224, 0.225]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: ['image', 'threshold_map', 'threshold_mask', 'shrink_map', 'shrink_mask'] # the order of the dataloader list
  loader:
    shuffle: True
    drop_last: False
    batch_size_per_card: 8
    num_workers: 4
    use_shared_memory: True

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./data
    label_file_list:
      - ./data/train_lpd_label.txt
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - DetLabelEncode: # Class handling label
      - DetResizeForTest:
          image_shape: [736, 1280]
      - NormalizeImage:
          scale: 1./255.
          mean: [0.485, 0.456, 0.406]
          std: [0.229, 0.224, 0.225]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: ['image', 'shape', 'polys', 'ignore_tags']
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: 1 # must be 1
    num_workers: 0
    use_shared_memory: True
````

**Comment by duong0411** [comment]
you need replace pretrain mobilenet to resnet 18 or resnet 50
